roles/1-prep/templates/iiab-expand-rootfs

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -x
+
+# Expand rootfs partition to its maximum size, if /.expand-rootfs exists.
+# Used by /etc/systemd/system/iiab-expand-rootfs.service on IIAB boot.
+
+# Should work with all Linux OS's boot disks -- regardless whether Raspberry Pi
+# microSD cards, external USB drives, internal spinning disks or SSD's, etc.
+
+# Verifies that rootfs is the last partition.
+
+if [ -f /.expand-rootfs ] || [ -f /.resize-rootfs ]; then
```

Comment:
Not sure why there would be a need to introduce another flag to act upon, all the tools in iiab-factory know nothing about the new 'expand-rootfs' flag.